### ISSUE TITLE
[Sentry] Add stats about devices distribution usage

### ISSUE THREAD

**Issue by dkijania** [input-output-hk/catalyst-toolbox#25]
Sentry log web client allows to dump mobile devices distribution. We can get information regarding registration:
```
{
    "message": "User registered with public_key: c53db2e7cd48bef1ca32f2e0c629118e5d1e4ab26d72af14e0463b57703569c6",
    "title": "User registered with public_key: c53db2e7cd48bef1ca32f2e0c629118e5d1e4ab26d72af14e0463b57703569c6",
    .
    .
    . 
    "user": {
      .
      .
      .
      "username": "c53db2e7cd48bef1ca32f2e0c629118e5d1e4ab26d72af14e0463b57703569c6",
    },
    "tags": [
      .
      .
      . 
      {
        "key": "device",
        "value": "SM-G991U"
      },
      {
        "key": "device.family",
        "value": "Samsung SM-G991U"
      },
      {
        "key": "environment",
        "value": "Production"
      },
      {
        "key": "os",
        "value": "Android 11"
      },
      {
        "key": "os.name",
        "value": "Android"
      },
    ],
  }, 
```

The problem is that it is almost impossible to print out distribution (in %) of above entry occuring between defined range. I would like to use catalyst-toolbox for that with query:

```
catalyst-toolbox logs sentry stats distribution --contains "User registered with public_key.*" --start-time  {DATE} --end-time {DATE}
```
where Date is in format: `"2021-06-15T07:11:25+00:00"`

**Comment by danielSanchezQ** [comment]
Covered in #26

**Comment by danielSanchezQ** [comment]
Implemented in #26
